pkgs/tools/text/uwuify/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, fetchFromGitHub, rustPlatform, ... }:
+
+let
+  version = "0.1.0";
+in
+rustPlatform.buildRustPackage {
+  pname = "uwu";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "Daniel-Liu-c0deb0t";
+    repo = "uwu";
+    rev = "v${version}";
+    sha256 = "0g0xc6133xxbcv19rbvzbpzgygsxkqxydc4v0mpnk97im6nskwhx";
+    fetchSubmodules = true;
+  };
+
+  cargoSha256 = "1v0aldxa31przlq2b6fwnl2grna69gbm33qiaj79cmfng43hyqsy";
```

Package works on Darwin too, but it needs an extra dependency.

```suggestion
  cargoSha256 = "1v0aldxa31przlq2b6fwnl2grna69gbm33qiaj79cmfng43hyqsy";
  
  buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [ libiconv ];
```